typescript/packages/well-log-viewer/src/utils/log-viewer.ts

```diff
@@ -19,6 +19,13 @@ export function isEqDomains(
     d1: Domain /*[number, number]*/,
     d2: Domain /*[number, number]*/
 ): boolean {
+    if (
+        Number.isNaN(d1[1]) &&
```

Comment:
did you mean || instead of && ?

Reply:
&&

Reply:
@Vladimir-Kokin Can you explain me why you do only return if ALL the 4 values are NaN, knowing that doing the computations below with any NaN does not make sense ?

Reply:
Usually no NaN values occure in a component run.
Only when some bogus data (null pointer and so on) is set to the component.
So I add only a check for the most frequent combination: [NaN,NaN], [NaN,NaN] to simplify some debugging.
There are still some another combinations with NaN: [NaN,n1], [NaN, n2] and so on but I do not check it in exact way for a sake of simplicity.